.github/workflows/docker-base.yaml

```diff
@@ -57,11 +61,12 @@ jobs:
           platforms: linux/amd64,linux/arm64,linux/arm/v7
           pull: true
           no-cache: true
-          push: true
+          push: github.event_name != 'pull_request'
```

Comment:
If this is the correct syntax then LGTM

Reply:
Should be correct as per https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request